quickstart/java-redisson-jcache/README.md

```diff
@@ -10,32 +10,33 @@ products:
 ---
 # Quickstart: Use Azure Cache for Redis in Java with Redisson Redis client and JCache
 
-This sample show you how to incorporate Azure Cache for Redis into a Java app using the [Redisson](https://redisson.org/) Redis client and [JCache](https://redisson.org/glossary/jcache.html).
+In this quickstart, you incorporate Azure Cache for Redis into a Java app using the [Redisson](https://redisson.org/) Redis client and JCP standard JCache API to have access to a secure, dedicated cache that is accessible from any application within Azure.
 
 ## Prerequisites
 
 - Azure subscription - [create one for free](https://azure.microsoft.com/free/)
-- Azure Cache for Redis cache - [create one](https://docs.microsoft.com/azure/azure-cache-for-redis/quickstart-create-redis)
 - [Apache Maven](https://maven.apache.org/download.cgi)
+- [Use Microsoft Entra ID for cache authentication](https://learn.microsoft.com/azure/azure-cache-for-redis/cache-azure-active-directory-for-authentication)
+
 
 ## Set up the working environment
 
-Depending on your operating system, add environment variables for your cache's **Host name** and **Primary access key**. Open a command prompt, or a terminal window, and set up the following values:
+Depending on your operating system, add environment variables for your cache's **Host name** and **USERNAME**. Open a command prompt, or a terminal window, and set up the following values:
 
 ```CMD
 set REDIS_CACHE_HOSTNAME=<YOUR_HOST_NAME>.redis.cache.windows.net
-set REDIS_CACHE_KEY=<YOUR_PRIMARY_ACCESS_KEY>
+set USERNAME=<USERNAME>
```

Comment:
We're entirely removing the option of using an access key? That raises the bar pretty high for a sample - now users will need to correctly configure Entra ID auth on their cache and client. Can we retain a code path for access keys too, or maybe leave some code comments indicating how to use them?

Reply:
Hello @philon-msft, thank you for your suggestion, I agree with you, and I will rework the samples today.

Reply:
Hi @philon-msft, I have updated the sample, I kept both auth with redis key and auth with Entra ID, thank you for your suggestion. Please tell me if you have any other concern.